README.md

```diff
@@ -31,12 +38,22 @@ requested (this is a multi-value alternative to DOMAINNAME)
 * `MQTT_RATE_LIMIT` - Enable rate limiting for MQTT connections (connections/s)
 
 ## Custom certificate format
+
 Any custom certificate volume mapped into `/etc/haproxy/certs` should be in PEM format and must include the full certificate chain and the private key, i.e.:
-```
- cat privkey.pem cert.pem chain.pem > ssl-certs.pem
+```shell
+cat privkey.pem cert.pem chain.pem > ssl-certs.pem
 ```
 
 See `haproxy` SSL cert [documentation](https://www.haproxy.com/blog/haproxy-ssl-termination/#enabling-ssl-with-haproxy).
 
 ## Edge gateway tunnelling using SISH
+
 The built in `haproxy.cfg` has support for forwarding requsts beginning with `gw-` to `https://SISH_HOST:SISH_PORT` just define these environment variables to enable this.
+
+## Kubernetes
+
+When running the proxy in Kubernetes make sure to set the `HTTP_PORT` and `HTTPS_PORT` environment variables to a non-privileged port (> 1024).
+If you use an Ingress, reconfigure the `HTTPS_FORWARDED_PORT` to the HTTPS port of your Ingress (443).
```

Comment:
I understand this as
if you have an Ingress in front of HAProxy, that lives on standard port 443 and you're accessing the manager through this Ingress, then you need to set HTTPS_FORWARDED_PORT

I think this applies to any means of accessing HAProxy. On EKS, I use a LB Service on 443 that maps to the HTTPS_PORT of HAProxy and I still set X-Forwarded-For to 443

Reply:
Yes I added this env var for this use case as I ran into the same issue with the Google Cloud Ingress. The `X-Forwarded-For` header is for instance used by Keycloak which returns responses with URLs based on this header.

The only change (besides the docs) with the version you were using is that the header is no longer hard coded now. I still needed to test if the default value (`%[dst_port]`) works correctly when set using an env var.

There are also Ingress controllers that can add these headers themselves so they could be passed on by the proxy. But from what I read not every Ingress controller adds these headers.